bundles/org.openhab.binding.vektiva/src/main/java/org/openhab/binding/vektiva/internal/config/VektivaSmarwiConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.vektiva.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link VektivaSmarwiConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Ondrej Pecta - Initial contribution
+ */
+@NonNullByDefault
+public class VektivaSmarwiConfiguration {
+
+    /**
+     * Smarwi configuration parameter.
+     */
+    public String ip = "";
```

Comment:
Is it really necessary that this is an IP address? I would guess that a hostname is allowed, too. If so, please rename to `host`.

Reply:
you are right, it might be host in the future, I'll consider renaming.